sdk/identity/Azure.Identity/src/MsalManagedIdentityClient.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.AppConfig;
+
+namespace Azure.Identity
+{
+    internal class MsalManagedIdentityClient
+    {
+        private readonly AsyncLockWithValue<IManagedIdentityApplication> _clientAsyncLock;
+        private bool _isForceRefreshEnabled { get; }
+
+        protected internal bool IsSupportLoggingEnabled { get; }
+        internal ManagedIdentityId ManagedIdentityId { get; }
+        protected internal bool DisableInstanceDiscovery { get; }
+        protected internal CredentialPipeline Pipeline { get; }
+        internal Uri AuthorityHost { get; }
```

Comment:
What is motivating some of these to be protected? I don't think I saw any class deriving from `MsalManagedIdentityClient`, in this PR.

Reply:
I don't think they actually need to be protected - these were previously only in MsalClientBase, but this class does not derive from it as the other msal clients do.